fdbrpc/AsyncFileNonDurable.actor.h

```diff
@@ -434,7 +434,8 @@ class AsyncFileNonDurable final : public IAsyncFile, public ReferenceCounted<Asy
 		state TaskPriority currentTaskID = g_network->getCurrentTask();
 		wait(g_simulator.onMachine(currentProcess));
 
-		state double delayDuration = deterministicRandom()->random01() * self->maxWriteDelay;
+		state double delayDuration =
+		    g_simulator.speedUpSimulation ? 0.0001 : deterministicRandom()->random01() * self->maxWriteDelay;
```

Comment:
This `0.0001` can be replaced with the new knob value?

Reply:
No, this is just saying be a fast disk if `speedupSimulation` is set, otherwise use a random portion of the `maxDelay` for this specific file, which itself is a random portion of the max value allowed by the knob, which is sometimes buggified to 5.